packages/gatsby-source-wordpress/src/steps/create-schema-customization/index.js

```diff
@@ -96,7 +98,12 @@ const createSchemaCustomization = async api => {
   try {
     await customizeSchema(api)
   } catch (e) {
-    api.reporter.panic(e)
+    api.reporter.panic({
+      id: CODES.SourcePluginCodeError,
+      context: {
+        sourceMessage: formatLogMessage(e.stack),
```

Comment:
I think it's helpful to also add `error: e` as a field in the `context` object.

Reply:
Oh wow, didn't realize it could do that! The output looks much nicer 👌 thank you!